### PR TITLE
replace to native methods in client code

### PR DESCRIPTION
**Pull request by miherlosev**
1)remove usage of a non native methods
2)remove redurant code (window.MozNamedAttrMap, Event.srcElement etc)
3)small refactoring

### CODE REVIEW

File: src/client/dom-processor/client-dom-adapter.js [hunk 1]
```diff
@@ -19,12 +19,7 @@ export default class ClientDomAdapter extends BaseDomAdapter {
     }
 
     hasAttr (el, attr) {
-        for (var i = 0; i < el.attributes.length; i++) {
-            if (el.attributes[i].name === attr)
-                return true;
-        }
-
-        return false;
+        return el.hasAttribute(attr);
```
Comment (churkin) [inline src/client/dom-processor/client-dom-adapter.js]: This was done to optimize. Please leave as it was and add a comment.

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 2]
```diff
@@ -91,9 +91,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             attributes: {
                 condition: el => {
-                    var attributesType = window.NamedNodeMap || window.MozNamedAttrMap;
-
-                    return attributesType && el.attributes instanceof attributesType;
+                    return el.attributes instanceof window.NamedNodeMap;
```
Comment (churkin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: `MozNamedAttrMap` is no longer relevant?

Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: yes

File: src/client/dom-processor/client-dom-adapter.js (hunk 1 above)
Comment (miherlosev) [inline src/client/dom-processor/client-dom-adapter.js]: It is wrong. 
see http://jsperf.com/has-attribute-versus-iterate (change to Chart Type to table view)

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR.
@churkin @kirovboris

**Comment by churkin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Pull request merged**